### PR TITLE
Wazo 1331 user line status

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: wazo_calld/plugins/endpoints/notifier.py [hunk 1]
```diff
@@ -2,33 +2,39 @@
...
 
-        event = TrunkStatusUpdatedEvent(body)
-        headers = {'tenant_uuid': trunk['tenant_uuid']}
+        trunk = self._confd_cache.get_trunk(endpoint.techno, endpoint.name)
+        if trunk:
+            body['id'] = trunk['id']
+            event = TrunkStatusUpdatedEvent(body)
+            headers = {'tenant_uuid': trunk['tenant_uuid']}
+
+        line = self._confd_cache.get_line(endpoint.techno, endpoint.name)
```
Comment (fblackburn1) [inline wazo_calld/plugins/endpoints/notifier.py]: Avoid hitting line cache if the result is already found in trunk

**Comment by pc-m** [comment]
> Can you standardize tests for bus events for trunk and line?

The tests in test_bus or in test_notifier? And what do you mean by standardize?

**Comment by pc-m** [comment]
Depends-On: https://github.com/wazo-platform/wazo-calld-client/pull/18

**Comment by pc-m** [comment]
Depends-On: https://github.com/wazo-platform/wazo-calld/pull/61

File: wazo_calld/plugins/endpoints/notifier.py (hunk 1 above)
Comment (fblackburn1) [reply wazo_calld/plugins/endpoints/notifier.py]: fixed

**Comment by bloom1** [comment]
@pc-m I think you have to add the depends-on in the PR description for them to work. Maybe @fredericlepied can confirm this, but this is what I read some time ago.

**Comment by fblackburn1** [comment]
Oh the rebase delete my latest commit that fixed documentation :disappointed:

**Pull request merged**